lib/features/favourite_user/ui/favourite_user_list_screen.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_userapp_demo/features/common/models/user.dart';
+import 'package:flutter_userapp_demo/features/common/widgets/user_card.dart';
+import 'package:flutter_userapp_demo/features/favourite_user/state/favourite_users_data.dart';
+import 'package:provider/provider.dart';
+
+class FavouriteUserListScreen extends StatelessWidget{
+  @override
+  Widget build(BuildContext context) {
+     return Scaffold(
+      appBar: AppBar(title: Text("Favourite Users"),),
+      body:FavouriteUserList(),
+    );
+  }
+}
+class FavouriteUserList extends StatelessWidget{
+  @override
+  Widget build(BuildContext context) {
+    var favUsers= Provider.of<FavouriteUsersData>(context);
+    List<User> favUsersData = favUsers.getFavUsers();
+    if(favUsersData.isEmpty) {
+      return Container(
+        width: MediaQuery.of(context).size.width,
+        color: Colors.white,
+        child: Column(
+          children:<Widget>[
+            SizedBox(
+              height: 100,
+            ),
+            Image.asset('assets/no_data.png',),
+            Text(
+               'No Favorites Found',
+                style: TextStyle(fontSize: 15),
+            )
+          ],
+        ),
+      );
+    }
```

Comment:
We can also subwidget this to make the Column more readable. You can create a sub widget 'BlankSlate' which will have this container.